### DISCUSSION TITLE
Clash analysis of elements

### DISCUSSION THREAD

**Discussion by ervinalla99** [IfcOpenShell/IfcOpenShell#4482] [Q&A] [answered]
Hello,
 I am relatively new to IfcOpenShell and I would like to know if anyone has managed to create a script for clash analysis within a specific distance.

**Comment by Moult** [comment, marked as answer]
Like this? https://blenderbim.org/docs-python/ifcopenshell-python/geometry_tree.html#detecting-clearance-clashes-between-elements

**Reply by ervinalla99** [reply]
Thank you for the reply. I want to use the script for the intersection clashes between the elements. I get some errors when I try to implement which I don't understand well due to my weak background in programming. 
I will try to solve them now that I have the sample.
